### PR TITLE
New package: squeezelite-1.9.9.1389

### PR DESCRIPTION
**Pull request by jpmenil**

### CODE REVIEW

File: srcpkgs/squeezelite/template [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# Template file for 'squeezelite'
+pkgname=squeezelite
+version=1.9.6.1198
+revision=1
+_githash=1c1f429ae9494748fe0fa5807c3a22710edaa12f
```
Comment (abenson) [inline srcpkgs/squeezelite/template:5]: Tagged releases are preferred.

Comment (ericonr) [reply srcpkgs/squeezelite/template:5]: They don't seem to tag stuff, eventhough versioned releases exist.

That said, seems like it's been updated to 1.9.7.1256 https://sourceforge.net/projects/lmsclients/files/squeezelite/linux/

I can't find which version was used to build those fragments, which is kind of suspect.

Comment (ericonr) [reply srcpkgs/squeezelite/template:5]: I think that works. Bad upstream, but it's what we got.

Comment (jpmenil) [reply srcpkgs/squeezelite/template]: i've update to the latest one. Can't do much with such upstream.

**Comment by jpmenil** [comment]
@ericonr anything i can do to have this one merged?

**Pull request closed without merging**